app/sidekiq/hca/log_email_diff_job.rb

```diff
@@ -28,5 +36,29 @@ def perform(in_progress_form_id, user_uuid)
       )
       $redis.set(redis_key, 't')
     end
+
+    def log_email_difference(in_progress_form_id, user_uuid)
+      return if FormEmailMatchesProfileLog.exists?(user_uuid:, in_progress_form_id:)
+
+      in_progress_form = InProgressForm.find_by(id: in_progress_form_id)
```

Comment:
I kept the same logic as the redis version here. If we do any refactors I plan to do those later just to keep the changes here scope to the database vs redis usage and not the logic in this method.